core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -1348,4 +1352,7 @@ void asyncStoreShutdownTimeoutTooHigh(long configuredAsyncStopTimeout,
 
    @Message(value = "Cannot remove cache configuration '%s' because it is in use", id = 371)
    IllegalStateException configurationInUse(String configurationName);
+
+   @Message(value = "Statistics are enabled while these are not available.", id = 372)
```

Comment:
It's not very clear from the error message that 'available' and 'enabled' are two different, and related, attributes.